### PR TITLE
Radix UI setup

### LINKED ISSUE #23: Radix UI setup
Now that react-bootstrap is not entirely compatible with server side rendering in the way that we would like, we should migrate to another UI library that does support SSR. Radix UI is one such library that we should try using.

### PR DESCRIPTION
**Pull request by samderanova**
- Installed Radix UI
- Set default Radix font to Poppins via `FontProvider`

### CODE REVIEW

**Comment by alexanderl19** [comment]
I forgot that Radix released a new themes system. I'll take a deeper dive into it later and see if it's compatible with Server Components.

File: apps/site/src/lib/theme/components/accordion/accordion.module.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.accordion-trigger {
```
Comment (samderanova) [inline apps/site/src/lib/theme/components/accordion/accordion.module.scss:1]: Nitpick [repeating]: we generally have uppercase file names (Accordion.module.scss, Button.module.scss) for components to distinguish them from other non-component files.

**Review by samderanova — COMMENTED** [review]
Thanks for contributing to this! The accordion does look somewhat plain, but the customizability will definitely benefit us in the long run. Feel free to approve whenever you're ready.

File: apps/site/src/lib/theme/components/accordion/accordion.module.scss (hunk 1 above)
Comment (alexanderl19) [reply apps/site/src/lib/theme/components/accordion/accordion.module.scss:1]: Thanks for catching this! The file name was an error.

**Comment by alexanderl19** [comment]
Thanks Sam! The accordion was just meant to be a placeholder to demonstrate how to style primitives. I'll fix the file name tomorrow and merge this.

Also I've run into quite a few instances where I think writing additional documentation would be helpful. After ZotHacks, could we explore some documentation solutions?

**Comment by samderanova** [comment]
> Thanks Sam! The accordion was just meant to be a placeholder to demonstrate how to style primitives. I'll fix the file name tomorrow and merge this.
> 
> Also I've run into quite a few instances where I think writing additional documentation would be helpful. After ZotHacks, could we explore some documentation solutions?

Yes, that would be helpful! I'll make a note to do this when the time comes.

**Comment by tyleryy** [comment]
Building off documentation, I feel like some single source of truth for past projects, practices, components would be great for future generations of developers on the Tech team. That way there is some reusability that can be used and we don't have to build everything from scratch.

File: apps/site/src/lib/theme/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		>
+			{children}
+		</Accordion.Trigger>
+	</Accordion.Header>
+));
+Trigger.displayName = Accordion.Trigger.displayName;
+
+const Content = React.forwardRef<
+	React.ElementRef<typeof Accordion.Content>,
+	React.ComponentPropsWithoutRef<typeof Accordion.Content> & {}
```
Comment (tyleryy) [inline apps/site/src/lib/theme/components/Accordion/Accordion.tsx:32]: For curiosity sake, what are you doing in this file?

**Review by tyleryy — APPROVED** [review]

**Pull request merged**